README.md

```diff
@@ -1,51 +1,61 @@
 # movie_radio
 simple radio page, that will play stream from Icecast server
 
-#####to install project on clean VM:
-(Python related stuff)
+###to install project on clean VM:
+#####Step 1 — Installing Ansible
+Add the Ansible PPA and refresh your system's package index, install the ansible
+by typing the following commands:
 
+```
+$ sudo apt-add-repository ppa:ansible/ansible
+$ sudo apt-get update
+$ sudo apt-get install ansible
+```
+
+#####Step 2 — Configuring Ansible Hosts
+Open the file with root privileges like this:
 
 ```
-1. sudo apt-get install git
-2. sudo apt-get install python-pip
-next line is for working python 3 terminal interpreter
-3. sudo apt-get install libreadline-dev
- SSL related dev libs
-4. sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libsqlite3-dev
-5. sudo apt-get install build-essential python-dev python-setuptools python-pip python-smbus
-6. sudo apt-get install libffi-dev
-7. Install python >=3.5 (see Install Python Section)
-8. pip install virtualenv
-9. virtualenv -p python3.5 env
-Activate virtual env
-10.source env/bin/activate
-11.pip install -r requirments.txt
+$ sudo vim /etc/ansible/hosts
 ```
-#####Install python(if not already installed)
-In case if you do not  have Python 3.5:
+
+Add the following configuration:
+
 ```
-    wget https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tar.xz
-    tar xf Python-3.5.1.tar.xz
-    cd Python-3.5.1
-    ./configure
-    make
-    make altinstall
+[barmaglot]
+your_server_ip
 ```
 
+Make sure that you have ssh connection to host that was specified
+and test connection via ansible:
+
+```
+$ ssh root@your_server_ip
```

Comment:
few remarks about this step.

 - It is possible that not all users know their `root` password. Basically I presume that user with `sudo` access is enough
 - in case of setting `127.0.0.1` as a `[barmaglot]` I got UNREACHEBLE error after ping command. This can be fixed by setting 
`[barmaglot]
127.0.0.1 ansible_connection=local`

I think this is some specific settings if you want to run Ansible locally, but maybe I am wrong :)